src/main/java/codeu/controller/AdminServlet.java

```diff
@@ -77,24 +83,31 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
         request.setAttribute("numConvo", numConvo);
         request.setAttribute("numUser", numUser);
         request.setAttribute("numMessages", numMessages);
-        request.setAttribute("lastUser", userStore.lastUser().getName());
+        request.setAttribute("lastUser", "foo");
```

Comment:
Can this be changed back to how it was before?

Reply:
Haha yeah this was a temporary fix because I didn't have the method pulled and I forgot to change it.